tick/optim/solver/build/__init__.py

```diff
@@ -1 +0,0 @@
-# License: BSD 3 clause
```

Comment:
This file must be kept for pip distribution (and moved to ` tick/solver/build/__init__.py`)